### ISSUE TITLE
[Enhancement] Add support for .avif images in cbr/cbz archives 

### ISSUE THREAD

**Issue by superuser-miguel** [YACReader/yacreader#425]
[AVIF files](https://en.wikipedia.org/wiki/AVIF) are supported in browsers, desktop etc. [Being open, royalty free format](https://aomediacodec.github.io/av1-avif/) . It has great compression of .jpg | .png files. I have been moving a collection over to the format and packing them in .cbz files for testing. On Linux, [**Gnome's Document reader evince**](https://gitlab.gnome.org/GNOME/evince) already supports .cbz files with .avif images and thumbnails can already be generated for them. gThumb also supports it OOTB. 

Seeing as this will be popular going forward, because of how av1 will be implemented by browsers and streeaming platforms, support for this can be drop in since the libraries are readily available.

**Comment by superuser-miguel** [comment]
The Fedora package has not been maintained since Fedora 33. . . 

https://koji.fedoraproject.org/koji/packageinfo?packageID=32052 

I would have to contact the developer on this. Thanks for the bit.

**Comment by selmf** [comment]
Yes I think installing kimageformats, either for Qt5 or Qt6 depending on how YACReader is built should do the trick.

**Comment by superuser-miguel** [comment]
That actually did not work, and Fedora packages avif support somewhere else. I do have Document Reader working just fine and have been using it for reading my newly converted files. I guess the flatpak would need to be packaged with it ?

**Comment by selmf** [comment]
Hmm, that is strange. Kimageformats should be the best way to get extra image format support for all Qt applications.

I did a quick check and it appears that it is installed in the newest edition of the KDE flatpak runtime. The YACReader flatpak is currently still on an older runtime, so updating this would probably make avif and other images available.

**Comment by nickv2002** [comment]
Bumping here, I'd love to see more AVIF & JXL support.

I specifically [docker](https://hub.docker.com/r/yacreader/yacreaderlibraryserver) `yacreader/yacreaderlibraryserver:9.15.0` but it doesn't seem to recognize CBZ archives with just AVIF images.
(I also have the lates iOS app for reading.)

**Comment by luisangelsm** [comment]
New plugins can be added now more easily using the settings folder, this applies to all the apps.

<img width="469" height="431" alt="Image" src="https://github.com/user-attachments/assets/bf328434-0031-458f-b737-c8c1c1787dfa" />
